recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "5.0.5" %}
+{% set version = "5.2.2" %}
```

Comment:
So, I think we are holding off on updating this at least until we can build everything (including Python) with a newer version of `xz`. Otherwise we have some breaks in the dependency chain.